model/stacktrace.go

```diff
@@ -7,12 +7,18 @@ import (
 
 type Stacktrace []*StacktraceFrame
 
-func (st *Stacktrace) Transform(config *pr.Config, service Service) []common.MapStr {
+func (st Stacktrace) Transform(config *pr.Config, service Service) []common.MapStr {
+	var fr *StacktraceFrame
 	var frames []common.MapStr
+	noFrames := len(st)
+	frames = make([]common.MapStr, noFrames)
 
-	for _, fr := range *st {
-		frame := fr.Transform(config, service)
-		frames = append(frames, frame)
+	fct := "<anonymous>"
```

Comment:
would be great to have the algorithm described here (or elsewhere) in comments, like you did in the PR description.

Reply:
Sure I can add a description. It would have been easier to follow if we applied it after the original sourcemapping was applied, but this would mean to walk through the whole stacktrace a second time.